core/tests/loadnext/src/config.rs

```diff
@@ -169,7 +169,7 @@ fn default_accounts_amount() -> usize {
 }
 
 fn default_duration_sec() -> u64 {
-    let result = 300;
+    let result = 400;
```

Comment:
why?

Reply:
Block times now increased from 1 second to 2 seconds and basically everything takes longer now

Reply:
correct but you cannot fit same 16000 txs into the different block time